packages/indexer-generator/src/render-domain.ts

```diff
@@ -258,10 +281,7 @@ export default class MainDomain
         account: meta.address,
         meta,
         index: {
-          transactions: {
-            chunkDelay: 0,
-            chunkTimeframe: 1000 * 60 * 60 * 24,
-          },
+          transactions: true,
```

Comment:
I guess `true` corresponds to some kind of default values?

Reply:
exactly